sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -88,6 +102,14 @@ internal class AmqpConnectionScope : IDisposable
         ///
         private static TimeSpan AuthorizationRefreshTimeout { get; } = TimeSpan.FromMinutes(3);
 
+        /// <summary>
+        ///   The amount of buffer to apply when considering an authorization token
+        ///   to be expired.  The token's actual expiration will be decreased by this
+        ///   amount, ensuring that it is renewed before it has expired.
+        /// </summary>
+        ///
+        private static TimeSpan AuthorizationTokenExpirationBuffer { get; } = AuthorizationRefreshBuffer.Add(TimeSpan.FromMinutes(2));
```

Comment:
I would make this longer than 2 minutes. In Azure.Identity we moved to a 5 minute buffer, and we have plans to make it half of token lifetime or 5 minutes whichever comes first.

Reply:
This ends up being "consider the token expired 10 minutes before its actual expiration date."     The refresh will start to ask for the token 8 minutes (plus ~30 seconds of random jitter) before it expires to ensure that the cached token is considered expired when the refresh takes place.

Reply:
Rebalanced to 7 and 3 to give a bit more disparity between them but keep the same 10 minute interval overall.